app/services/document_processor.py

```
@@ -330,8 +388,8 @@ async def _process_pdf(
 
             # If no text was extracted, PDF might be scanned - log warning
             if len(result["content"]["text"].strip()) < 50:
-                logger.warning(f"Very little text extracted from PDF - might be scanned. Consider OCR.")
-                result["errors"].append("Low text extraction - PDF may be scanned/image-based")
+                logger.warning(f"Very little text extracted from PDF - might be scanned. LlamaParse Premium recommended.")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)